### PR TITLE
fixed determineJava11OrLater

### PR DESCRIPTION
**Pull request by cdietrich**
Signed-off-by: Christian Dietrich <christian.dietrich@itemis.de>

### CODE REVIEW

File: org.eclipse.xtext.xbase.tests/src/org/eclipse/xtext/xbase/tests/AbstractXbaseTestCase.java [hunk 1]
```diff
@@ -41,8 +43,13 @@ public static boolean isJava11OrLater() {
 	private static boolean determineJava11OrLater() {
 		String javaVersion = System.getProperty("java.version");
 		try {
-			int version = Integer.parseInt(javaVersion);
-			return version >= 11;
+			Pattern p = Pattern.compile("(\\d+)(.)*");
```
Comment (szarnekow) [inline org.eclipse.xtext.xbase.tests/src/org/eclipse/xtext/xbase/tests/AbstractXbaseTestCase.java]: I wonder if we should move this to xtext.util.JavaVersion?

Comment (cdietrich) [reply org.eclipse.xtext.xbase.tests/src/org/eclipse/xtext/xbase/tests/AbstractXbaseTestCase.java]: methodname?

Comment (szarnekow) [reply org.eclipse.xtext.xbase.tests/src/org/eclipse/xtext/xbase/tests/AbstractXbaseTestCase.java]: JavaVersion.getRuntimeVersion(): JavaVersion?

Comment (cdietrich) [reply org.eclipse.xtext.xbase.tests/src/org/eclipse/xtext/xbase/tests/AbstractXbaseTestCase.java]: well then i have to detect 1.4-1.8 too

Comment (szarnekow) [reply org.eclipse.xtext.xbase.tests/src/org/eclipse/xtext/xbase/tests/AbstractXbaseTestCase.java]: Hmm true. Not worth it as of now.

Comment (cdietrich) [reply org.eclipse.xtext.xbase.tests/src/org/eclipse/xtext/xbase/tests/AbstractXbaseTestCase.java]: will file a second issue, maybe you can give it a shot

**Pull request merged**